src/pymodaq_plugins_cellkraft/daq_move_plugins/daq_move_CellkraftE1500.py

```diff
@@ -88,30 +88,23 @@ class DAQ_Move_CellkraftE1500(DAQ_Move_base):
     def ini_attributes(self):
         self.controller: CellKraftE1500Drivers
         self.current_axes = self.settings.child('multiaxes', 'axis').value()
-        self.change_param()
-        pass
-
-    def change_param(self):
-
-        if self.current_axes == 'Flow':
-            self.settings.child('info').setValue(self.desc['Flow'])
-            self.settings.child('limit').setValue(self.lim['Flow'])
-
-        elif self.current_axes == 'Steam_Temperature':
-            self.settings.child('info').setValue(self.desc['Steam_Temperature'])
-            self.settings.child('limit').setValue(self.lim['Steam_Temperature'])
-
-        elif self.current_axes == 'Tube_Temperature':
-            self.settings.child('info').setValue(self.desc['Tube_Temperature'])
-            self.settings.child('limit').setValue(self.lim['Tube_Temperature'])
-
-        elif self.current_axes == 'RH':
-            self.settings.child('info').setValue(self.desc['RH'])
-            self.settings.child('limit').setValue(self.lim['RH'])
 
-        elif self.current_axes == 'Pressure':
-            self.settings.child('info').setValue(self.desc['Pressure'])
-            self.settings.child('limit').setValue(self.lim['Pressure'])
+        match self.current_axes:
+            case 'Flow':
+                self.settings.child('info').setValue(self.desc['Flow'])
+                self.settings.child('limit').setValue(self.lim['Flow'])
+            case 'Steam_Temperature':
+                self.settings.child('info').setValue(self.desc['Steam_Temperature'])
+                self.settings.child('limit').setValue(self.lim['Steam_Temperature'])
+            case 'Tube_Temperature':
+                self.settings.child('info').setValue(self.desc['Tube_Temperature'])
+                self.settings.child('limit').setValue(self.lim['Tube_Temperature'])
+            case 'RH':
+                self.settings.child('info').setValue(self.desc['RH'])
+                self.settings.child('limit').setValue(self.lim['RH'])
+            case 'Pressure':
+                self.settings.child('info').setValue(self.desc['Pressure'])
+                self.settings.child('limit').setValue(self.lim['Pressure'])
```

Comment:
Why not replacing the whole mach/case block using self.current_axes as variable ?
```suggestion
        self.settings.child('info').setValue(self.desc[self.current_axes])
        self.settings.child('limit').setValue(self.lim[self.current_axes])
```